kolibri/plugins/learn/assets/src/vue/learn-page/allcontent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+
+  <card-grid :header="'All Content'" v-if="all.content.length">
+
+    <div slot="headerbox" class="allnav">
+
```

Comment:
If the `id` for headings is a no-go for now, we can always put `aria-label="Browse all content"` instead of `aria-labelledby`.